### PR TITLE
Removed vuetify components from index.vue

### PR DESCRIPTION
**Pull request by shukapurv**
### Removed vuetify components and used tailwind classes and HTML tags instead
Screenshot 2021-06-04 at 10 03 42 PM

### CODE REVIEW

**Comment by KKVANONYMOUS** [comment]
@shukapurv There are some conflicts in your PR. I guess these conflicts are due to your fork being not even with the main branch. So, update your fork accordingly and then create a new PR with the proposed changes.

**Comment by shukapurv** [comment]
@KKVANONYMOUS I tried to resolve the conflicts, I guess now there is no conflict in my PR.
Should i create new PR or is it okay?

File: pages/index.vue [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
+                  hover:tw-scale-110
+                  tw-transform tw-cursor-pointer
+                "
+                :data-icon="item.icon"
+                data-inline="false"
+              >
+              </span>
+            </div>
+          </div>
+        </div>
```
Comment (KKVANONYMOUS) [inline pages/index.vue:32]: No need of social media links here. Already added into footer

File: pages/index.vue [hunk 2]
```diff
@@ -53,8 +69,20 @@ export default {
...
+        { icon: 'mdi:email', url: 'mailto:copsiitbu@gmail.com' },
+        { icon: 'mdi:rss', url: 'https://copsiitbhu.co.in/feed.xml' },
+      ],
     }
   },
+  methods: {
+    iconClick(a) {
+      window.location.href = a.url
+    },
+  },
```
Comment (KKVANONYMOUS) [inline pages/index.vue:85]: No need of this also

File: pages/index.vue [hunk 3]
```diff
@@ -53,8 +69,20 @@ export default {
             'Although the club has been functioning quite well as compared to other clubs, still, a lot of things needs to be done. Some great initiatives were taken by the current secretaries, including the formation of smaller groups focussed on specific areas (competitive programming group, development group and machine learning group) and the fresher’s mentorship program. Still there are lot of things that the club can improve upon. We hope that we are able to take this momentum forward.',
         },
       ],
+      items: [
+        { icon: 'mdi:github', url: 'https://github.com/COPS-IITBHU' },
+        { icon: 'mdi:facebook', url: 'https://www.facebook.com/cops.iitbhu' },
+        { icon: 'mdi:google', url: 'https://groups.google.com/g/copsiitbhu' },
+        { icon: 'mdi:email', url: 'mailto:copsiitbu@gmail.com' },
+        { icon: 'mdi:rss', url: 'https://copsiitbhu.co.in/feed.xml' },
+      ],
```
Comment (KKVANONYMOUS) [inline pages/index.vue:78]: This can be removed as icons are already there in footer

**Review by KKVANONYMOUS — CHANGES_REQUESTED** [review]
One more thing, instead of using `div` everywhere you can make use of semantic HTML to increase the readability of the code

**Pull request closed without merging**